daemon/src/model/cfd.rs

```diff
@@ -740,6 +741,11 @@ pub struct Dlc {
     /// The fully signed lock transaction ready to be published on chain
     pub lock: (Transaction, Descriptor<PublicKey>),
     pub commit: (Transaction, EcdsaAdaptorSignature, Descriptor<PublicKey>),
-    pub cets: Vec<(Transaction, EcdsaAdaptorSignature, RangeInclusive<u64>)>,
+    pub cets: CETs,
     pub refund: (Transaction, Signature),
 }
+
+type CETs = HashMap<
+    Vec<schnorrsig::PublicKey>,
+    Vec<(Transaction, EcdsaAdaptorSignature, RangeInclusive<u64>)>,
+>;
```

Comment:
Stupid GitHub added my comment somewhere else.

I was meant to comment here:

If we have a hashmap here, why did we need a different structure within the protocol? :thinking:

Reply:
I first thought so as well, but there is a slight difference.
In the protocol we have 
`Vec<(Transaction, EcdsaAdaptorSignature, interval::Digits)>,`
while here we have
`Vec<(Transaction, EcdsaAdaptorSignature, RangeInclusive<u64>)>,`